src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -630,16 +631,6 @@ export const selectCryptoHumanBalanceIncludingStakingByFilter = createCachedSele
   genericBalanceIncludingStakingByFilter,
 )((_s: ReduxState, filter) => `${filter?.accountId}-${filter?.assetId}` ?? 'accountId-assetId')
 
-export const selectPortfolioChainIdsSortedUserCurrency = createDeepEqualOutputSelector(
-  selectPortfolioAccountsUserCurrencyBalancesIncludingStaking,
-  (userCurrencyAccountBalances): ChainId[] =>
-    Array.from(
-      new Set(
-        Object.keys(userCurrencyAccountBalances).map(accountId => fromAccountId(accountId).chainId),
-      ),
-    ),
-)
-
```

Comment:
This appears to be creating an ordered set of chainIds using the `userCurrencyAccountBalances`. The issue here is the the most recent entry will override a previous one, meaning the following issues could arise:
- lots of small balance accounts on a chain with a total higher than 1 large balance on a different chain would be ordered lower
- a large balance followed by a low balance would be ordered lower than a different chain with a moderately sized balance

Reply:
The solution proposed in this PR is to sum all of the account balances and sort them based on the total.